pkg/tsdb/azuremonitor/loganalytics/azure-log-analytics-datasource_test.go

```diff
@@ -1535,8 +1535,11 @@ func Test_executeQueryErrorWithDifferentLogAnalyticsCreds(t *testing.T) {
 		TimeRange: backend.TimeRange{},
 	}
 	tracer := tracing.InitializeTracerForTest()
-	_, err := ds.executeQuery(ctx, query, dsInfo, &http.Client{}, dsInfo.Services["Azure Log Analytics"].URL, tracer)
-	if !strings.Contains(err.Error(), "credentials for Log Analytics are no longer supported") {
+	res := ds.executeQuery(ctx, logger, query, dsInfo, &http.Client{}, dsInfo.Services["Azure Log Analytics"].URL, tracer)
+	if res.Error == nil {
+		t.Fatal("expecting an error")
+	}
+	if !strings.Contains(res.Error.Error(), "credentials for Log Analytics are no longer supported") {
```

Comment:
This also looks like an unintended change - undoing the changes from the latest commit to this file.